src/Header.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+const Header = (props) => {
+    return (
+        <header className="header">
+            <h1>todos</h1>
+
+            <input
+                className="new-todo"
+                placeholder="What needs to be done?"
+                autoFocus=""
+                onKeyDown={(event) => {
+                    if (event.key === 'Enter') {
+                        const newItem = { 'text': event.target.value, 'isCompleted': false };
```

Comment:
Add `id` field. It is mandatory to have ID and it should be unique for each item. 
Suggestion: use `Date.now()` as id
Suggestion 2: use `random string` library: https://www.npmjs.com/package/randomstring